src/RpcServer/RpcError.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// The Neo.Network.RPC is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.Json;
+
+namespace Neo.Plugins
+{
+    public class RpcError
+    {
+        #region Default Values
+
+        //public static readonly RpcError InternalServerError = new(-32603, "Internal server RpcError");
+        public static readonly RpcError BadRequest = new(-32700, "Bad request");
+        public static readonly RpcError InvalidRequest = new(-32600, "Invalid request");
+        public static readonly RpcError MethodNotFound = new(-32601, "Method not found");
+        public static readonly RpcError InvalidParams = new(-32602, "Invalid params");
+
+        public static readonly RpcError UnknownBlock = new(-101, "Unknown block");
+        public static readonly RpcError UnknownContract = new(-102, "Unknown contract");
+        public static readonly RpcError UnknownTransaction = new(-103, "Unknown transaction");
+        public static readonly RpcError UnknownStorageItem = new(-104, "Unknown storage item");
+        public static readonly RpcError UnknownStateRoot = new(-105, "Unknown state root");
```

Comment:
@shargon, please, pay attention that codes should be taken from the resulting NEP: https://github.com/neo-project/proposals/pull/156, not from the initial issue (https://github.com/neo-project/neo-modules/issues/728). In the NEP some codes were adjusted.

`UnknownStateRoot` is `-106`.